app/src/main/java/com/codesoom/assignment/controllers/UserController.java

```diff
@@ -10,6 +10,9 @@
 
 import javax.validation.Valid;
 
+/***
+ * 사용자 세션 관련 컨트롤러
+ */
```

Comment:
컨트롤러를 설명할 때 컨트롤러라고 할 필요는 없습니다. 이 클래스의 책임 하나만 단순하게 설명해 줍시다.
```suggestion
/***
 * 사용자 세션 관련 http 요청을 처리합니다.
 */
```